redfish_client/resource.py

```diff
@@ -86,10 +86,16 @@ def __getattr__(self, name):
         return self[name]
 
     def __getitem__(self, name):
-        return self._build(self._content[name])
+        return self._build(self._get_content()[name])
 
     def __contains__(self, item):
-        return item in self._content
+        return item in self._get_content()
+
+    def _get_content(self):
+        oid = self._content.get("@odata.id")
+        if oid and len(self._content) == 1:
```

Comment:
What do you think about having an explicit flag (e.g., `self._stub`) rather than learning indirectly from content to obtain the status of the resource?